src/components/PaymentsForm/PaymentForm.tsx

```diff
@@ -61,37 +61,41 @@ const PaymentsForm = () => {
                   ? "Number of account for payment: xx xxxx xxxx xxxx xxxx"
                   : "Tytuł przelewu: WTYCZKA2023-ImieNazwisko  Kwota: xxx zł "}
               </Typography>
-              <div className="grid grid-cols-1 gap-1 justify-center items-center  md:min-w-[30rem]">
-                <div className="xl:flex-row gap-1 flex flex-col">
-                <FormField
-                  label={languageMode == "english" ? "Name" : "Imię"}
-                  isRequired={false}
-                  minLength={3}
-                  maxLength={30}
-                  registerName="name"
-                />
-                <FormField
-                  label={languageMode == "english" ? "Surname" : "Nazwisko"}
-                  isRequired={false}
-                  minLength={3}
-                  maxLength={30}
-                  registerName="surname"
-                />
+              <div className="grid grid-cols-1 gap-1 justify-center items-center w-full sm:w-5/6 xl:w-1/2">
+                <div className="flex gap-1 w-full max-xl:flex-col">
+                  <div className="w-1/2 max-xl:w-full">
+                    <FormField
+                      label={languageMode == "english" ? "Name" : "Imię"}
+                      isRequired={true}
+                      minLength={3}
+                      maxLength={30}
+                      registerName="name"
+                    />
+                  </div>
+                  <div className="w-1/2 max-xl:w-full">
+                    <FormField
+                      label={languageMode == "english" ? "Surname" : "Nazwisko"}
+                      isRequired={true}
+                      minLength={3}
+                      maxLength={30}
+                      registerName="surname"
+                    />
+                  </div>
                 </div>
```

Comment:
To be honest i do not understand why there are these third div's(the ones that wraps FormFIeld).

Reply:
This is to manipulate the space input takes. If form components would have className I would add this styling to them. We had a problem where these two fields were a lot smaller when going to mobile than other fields below it. That takes care of it

Reply:
Before
![image](https://github.com/WRS-WEEIA-IT/WTYCZKA-2023/assets/106811447/66cd639b-c0b4-4df5-938d-974366e9efb0)
After
![image](https://github.com/WRS-WEEIA-IT/WTYCZKA-2023/assets/106811447/7ba29eac-270f-4e75-8f3b-8eb8c17d5a1a)
I will add custom className to field to make it look better